.circleci/config.yml

```diff
@@ -288,7 +288,7 @@ jobs:
       - run:
           name: test op-e2e
           command: |
-            gotestsum --junitfile /test-results/op-e2e.xml -- -coverpkg=github.com/ethereum-optimism/optimism/... -coverprofile=coverage.out -covermode=atomic ./...
+            gotestsum --format standard-verbose --junitfile /test-results/op-e2e.xml -- -coverpkg=github.com/ethereum-optimism/optimism/... -coverprofile=coverage.out -covermode=atomic ./...
```

Comment:
Should it be `-coverpkg=github.com/ethereum-optimism/optimism/op-e2e/...` ?

Reply:
Should be Optimism, since we're covering other Optimism packages with these tests.